app/assets/stylesheets/framework_and_overrides.css.scss

```diff
@@ -43,14 +43,42 @@
   @extend .button;
   @extend .radius;
   }
+
+.simple_form {
+  background-color: rgba(255,255,255,0.7);
+  padding: 5%;
+  border-radius: 10px;
+}
 // apply styles to HTML elements
 // to make views framework-neutral
+
 main {
   @include twelve-columns;
+  background: url('TremontStreet-blur.jpg') no-repeat center center fixed;
+  background-size: cover;
```

Comment:
Properties should be ordered background, background-color, background-size, height, padding-top